.gitignore

```diff
@@ -1 +1,2 @@
 .idea
+node_modules
```

Comment:
node_modules and node_modules/ are almost the same.

Reply:
sorry, i do not understend.

Reply:
Read this article: [gitignore](https://www.atlassian.com/ru/git/tutorials/saving-changes/gitignore), and look at the task carefully.